SEED-platform/seed
# #2437 - feat(analysis): create model and migration

#### Any background context you want to provide?
Part of the Analysis "Epic"

#### What's this PR do?
Create Analysis Django Model and corresponding migration.

#### How should this be manually tested?
Should be able to run migration. Monkey test Analysis object in shell, ensure type and name are required.

#### What are the relevant tickets?
#2437 

#### Screenshots (if appropriate)
